db/migrate/20200914005000_add_version_and_cloud_provider_to_configuration_profiles.rb

```diff
@@ -0,0 +1,6 @@
+class AddVersionAndCloudProviderToConfigurationProfiles < ActiveRecord::Migration[5.2]
+  def change
+    add_column :configuration_profiles, :default_version, :string
```

Comment:
I wonder if it'd be better to collect all versions and have a "default" boolean

Reply:
Removed the versions column from the existing PR. Will create a separate one when we have a better solution for representing the template versions in manageig